### DISCUSSION TITLE
Empty-ish Android notification

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#46244] [Mobile]
GitHub Mobile beta

I'm not sure if this was supposed to be an auth prompt or an event notification, but it's void:
![Screenshot_20230202-190311.png](https://user-images.githubusercontent.com/2119212/216535698-d7381384-e1b5-48a8-b1eb-91e4ca0c8e66.png)

![Screenshot_20230202-190339.png](https://user-images.githubusercontent.com/2119212/216535710-391a086e-444b-4b61-b933-a047f810177c.png)

Expected results: don't do this. Either show text or revoke your record.

**Comment by alcere** [comment]
Hi @jsoref, thanks for reporting.
This should not be happening. Do you have any idea or hint on what triggered this notification? For example, did you receive it during a 2FA authentication process while logging in using the browser?

**Reply by jsoref** [reply]
I'm really not certain. It's possible to was that. I did do a number of those actions last night. 

There was also one time where it opened an alert saying there weren't any sudo operations, and then shortly thereafter showed the proper dialog. Sorry, I was stunned and also trying to resolve that specific sudo request...

**Reply by jsoref** [reply]
Fwiw, the notification is still there. It was from 14h ago. Would it help for me to open a ticket so someone could try to find the event?

**Reply by alcere** [reply]
> Fwiw, the notification is still there. It was from 14h ago. Would it help for me to open a ticket so someone could try to find the event?

Yes please, it won't hurt!

**Reply by jsoref** [reply]
For my reference: https://support.github.com/ticket/personal/0/2000326
